src/tests/Microsoft.PowerFx.Interpreter.Tests/ValueTests.cs

```diff
@@ -358,6 +359,23 @@ public void NewError()
             Assert.All(combinedError.Errors, (e) => e.Kind = ErrorKind.Custom);
             Assert.Equal("test1", combinedError.Errors.First().Message);
         }
+
+        [Fact]
+        public void NewRecordFromFieldsTest()
```

Comment:
We should just add a comment like:
// ensure the enumeration is only traversed once.

Reply:
add comment